spec/enumerable_spec.rb

```diff
@@ -0,0 +1,224 @@
+#spec/enumerable_spec.rb
+require './lib/enumerable'
+
+RSpec.describe Enumerable do
+  test = [2,3,4,5,6,7]
+  testeven = [2,4,6,8,10]
+  testodd = [1,3,5,7,9]
+  hash = {"one" => 1, "two" => 2, "three" => 3}
+  testclass = Class.new
```

Comment:
- [ ] it's recommended to declare the variables like;
`let(:test){[2,3,4,5,6,7]},
  let(:testeven){[2,4,6,8,10]},
  let(:testodd){[1,3,5,7,9]},
  let(:hash){{"one" => 1, "two" => 2, "three" => 3}},
  let(:testclass){Class.new}`